### DISCUSSION TITLE
Cloudflare IPv4/IPv6

### DISCUSSION THREAD

**Discussion by bonelifer** [qdm12/ddns-updater#344] [Q&A] [answered]
How would the config file look for updating both? Do I need to have two entries, one for IPv4 and IPv6?

https://github.com/qdm12/ddns-updater/blob/master/docs/cloudflare.md

**Comment by tcurdt** [comment, marked as answer]
It seems for the moment you will have to use two entries: https://github.com/qdm12/ddns-updater/issues/507

Unfortunately :-(
